mock/api.js

```diff
@@ -13,34 +13,19 @@ export const mockControllers = new Array(DEFAULT_SIZE).fill().map((value, index)
   results: casual.integer(1, DEFAULT_SIZE),
 }));
 
-export const mockResults = {
-  took: DEFAULT_SIZE,
-  hits: {
-    total: {
-      value: DEFAULT_SIZE,
-    },
-    hits: new Array(DEFAULT_SIZE).fill().map((value, index) => ({
-      _index: moment().format('YYYY-MM'),
-      _type: casual.word,
-      _id: casual.uuid,
-      _score: null,
-      _source: {
-        '@metadata': {
-          controller_dir: casual.word,
-        },
-        run: {
-          controller: casual.word + index,
-          name: casual.word,
-          start: moment.utc() + index,
-          end: moment.utc() + index,
-          id: casual.uuid,
-          config: casual.word,
-        },
-      },
-      sort: [casual.unix_time],
-    })),
-  },
-};
+export const mockResults = new Array(DEFAULT_SIZE).fill().map((value, index) => ({
+  '@metadata.controller_dir': casual.word,
+  key: casual.uuid,
+  id: casual.uuid,
+  config: casual.word,
+  controller: casual.word + index,
+  deletion: moment.utc().add('days', 30),
+  end: moment.utc() + index,
+  result: casual.word,
+  seen: false,
+  saved: Math.random() < 0.5,
+  status: 'published',
```

Comment:
What other values do you propose for `status`?? In the Elasticsearch documents, I've defined `owner` and `access` fields, where access will start out as `private` and can be changed to `public` by publishing it. I chose to represent it that way to support eventual expansion to groups of users, such that the value of `access` might in the future be a group name defining a set of users who can access it.

Along the same lines, we might have `access` here with a direct pass-through of the document access. Or we could keep it simple and for now just use `published: true | false`. In any case, again, I don't think we should be expanding the mock in this PR; just fix it. We should discuss how we want to change the query going forward, and then define issues for both the dashboard and server to implement.

Reply:
I am removing the context of this new `access` or `status` field from this PR. These will be referenced in #105